qiita_pet/templates/analysis_selected.html

```diff
@@ -63,6 +72,10 @@
 </div>
 {% end %}
 
+<button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#create-analysis-modal-view">
```

Comment:
I have it at the bottom to force people to look at what they selected before hitting create. I can make it green though.